SnyderMbishai/fast_food_fast_backend
# [# 160788989] Set Up Database

#### What this PR does:
This PR sets up the database, 
uses blueprints to separate the versions, 
connects tests to the test_db database, 
connects models to the development database.

#### Task description:
Set up database.
Create tables.
Create models and their methods.
Set up the base test to connect to db.

#### Manual testing:
$ virtualenv venv
$ cd venv
$ source bin/activate
$ git clone https://github.com/SnyderMbishai/fast_food_fast_backend.git
$ cd fast_food_fast_backend
$ git checkout ft-set-up-db-160788989
$ export APP_SECRET_KEY="sommethingrandom"
$ export APP_SETTINGS="development"
$ export PORT=5000
$ export TESTING_DB="test_db"
$ export DEV_DB="development"
$ export USER="your postgres"
$ export PASSWORD="yourpassword"
$ export HOST="localhost"
$ python run.py
Make sure you have created the databases locally


#### PT story:
160788989
